### ISSUE TITLE
<Field /> & <FastField /> should have validateOnChange & validateOnBlurSettings

### ISSUE THREAD

**Issue by maxmantz** [jaredpalmer/formik#1044]
### Current Behavior

`<FastField />` & `<Field />` don't have `validateOnChange` & `validateOnBlur` settings.

### Expected behavior

Setting `validateOnChange` & `validateOnBlur` on `<Field />` or `<FastField />` should override global form settings.

### Reproducible example

https://codesandbox.io/s/3q55883491

### Suggested solution(s)

Make props `validateOnChange` & `validateOnBlur` available for `<Field />` & `<FastField />`.

### Additional context

This may or may not be a bug. I'm using Typescript and I don't get an error message when I set `validateOnBlur` or `validateOnChange` on `<FastField />`. Does this mean these props already exist, but don't work?

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Formik           | 1.3.1
| React            |  16.5.0 |
| TypeScript       | 3.0.0 |
| Browser          |  Chrome |

**Comment by c10b10** [comment]
Ping.

**Comment by dunk** [comment]
Yes, I could really do with this, too. For instance: on a normal text input you likely want validation `onBlur`, but with a select or a radio button `onChange` is much more appropriate.

**Comment by Ericnr** [comment]
Bump for this, would be nice to validate password/confirm password to be the same `onChange` and other fields after submit only

**Comment by shandysulen** [comment]
Another bump for this. Has there been any progress regarding this enhancement?

**Comment by youngchingjui** [comment]
Hello, yes adding another bump here.

My use case: On account registration, we have following basic fields:
1. Name
2. Company
3. Email
4. Password
5. Confirm password

We'd like to validate the form on submit for fields 1 - 3 & 5.
We'd like to validate the password field with onChange, so we can show a dynamic password strength meter.

**Comment by developer-collin** [comment]
Would find this feature very useful as well. Use case being similar to others.

For text-like inputs validation onBlur is often a better UX than onChange, as changing errors states mid-typing can be jarring. Similarly, onBlur in those cases showcase user-intention to be done, so validation at that point makes sense.

For checkboxes and radios, we want validation onChange, as we know the user interaction is already complete, yet onBlur may not take place in a timely manor, leaving a user without feedback.
